### ISSUE TITLE
#Help #Question How does one actually create text in the tab?

### ISSUE THREAD

**Issue by ldexterldesign** [hayden-t/textTab#3]
Hey,

Thanks for software

How does one actually create text in the tab?

Hope to hear back

Kind thanks

**Comment by hayden-t** [comment]
Hey, you create text file, and specify the path to that file in the settings for the plugin. Let me know how you go, i have not yet coded the settings through web interface.

**Comment by ldexterldesign** [comment]
Cheers for reply

How/where can I access the plugin settings?

Hope to hear back

Kind thanks

**Comment by hayden-t** [comment]
Are you using the application version of deluge ?I have not coded the settings screen in the web panel.

**Comment by ldexterldesign** [comment]
You haven't really answered my question

I'm able to use both desktop and web versions of Deluge, but, in either case, I can't work out the solution 😕?

Hope to hear back

Kind thanks

**Comment by hayden-t** [comment]
correct, i see now it is not installing, nor is any message being generated about that in the debug log. this is going to have to be investigated more, sorry for confusion

**Comment by hayden-t** [comment]
i believe you are using the wrong version of deluge, this plugin is made for the 2.7 python version: http://download.deluge-torrent.org/windows/py2.7/

**Comment by ldexterldesign** [comment]
I'm using Python v2.7.11 in OS X

Can you confirm your plugin is for Windows?

Thanks

**Comment by hayden-t** [comment]
i have only tested on windows 
![capture](https://cloud.githubusercontent.com/assets/1000743/16937157/dcc9935e-4daf-11e6-84ab-b7bbeada29c6.JPG)
